provisioning/transport/amqp/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "azure-iot-provisioning-amqp",
+  "name": "azure-device-provisioning-amqp",
```

Comment:
>azure-device-provisioning-amqp [](start = 11, length = 30)

I think it should be azure-provisioning-device-client

Reply:
really? "Azure provisioning" carries a very different meaning than "azure device provisioning". I'm really not sure about that one...

---
In reply to: [147511225](https://github.com/Azure/azure-iot-sdk-node/pull/126#discussion_r147511225) [](ancestors = 147511225)